src/features/world/ui/factions/emblemTrading/Trade.tsx

```diff
@@ -372,6 +376,51 @@ const TradeDetails: React.FC<{
   );
 };
 
+const ISLAND_LIMITS: Record<IslandType, number> = {
+  basic: 5,
+  spring: 10,
+  desert: 20,
+};
+
+export function getRemainingListings({ game }: { game: GameState }) {
```

Comment:
So the listings limit is coming back with the new system? Bummer. Probably should be formally announced. :|

Reply:
No, these files will be removed 👀 It is now legacy code - we are just leaving it in here in case we need a quick rollback on certain sides.